### PR TITLE
telegram: fix broken links, add setup instructions

### PR DESCRIPTION
**Pull request by half-duplex**
##### Summary
Fix, add, and update documentation and comments for telegram alert configuration. Replace edited image.

##### Component Name
health/notifications/telegram/README.md

### CODE REVIEW

File: health/notifications/telegram/README.md [hunk 1]
```diff
@@ -9,13 +9,31 @@ Netdata will send warning messages without vibration.
...
+```
+role_recipients_telegram[sysadmin]="212341234"
+role_recipients_telegram[dba]="-1004444333321"
+role_recipients_telegram[webmaster]="49999333322 -1009999222255"
+```
 
 Telegram messages look like this:
 
-![image](https://fb.hash.works/ytl/preview.jpg)
+![image](https://i.imgur.com/5AvoSI5.png)
```
Comment (joelhans) [inline health/notifications/telegram/README.md]: We prefer our images be hosted by GitHub's CDN. Here's the same image over there (plus an alt 😉):

```
![Netdata notifications via Telegram](https://user-images.githubusercontent.com/1153921/66612223-f07dfb80-eb75-11e9-976f-5734ffd93ecd.png)
```

Comment (half-duplex) [reply health/notifications/telegram/README.md]: Thanks!

File: health/notifications/telegram/README.md [hunk 2]
```diff
@@ -9,13 +9,31 @@ Netdata will send warning messages without vibration.
 You need:
 
 1.  A bot token. To get one, contact the [@BotFather](https://t.me/BotFather) bot and send the command `/newbot`. Follow the instructions.
-2.  A chat id for every chat you want to send messages to. Contact the [@myidbot](https://t.me/myidbot) bot and send the command `/getid` to get your personal chat id or invite him into a group and issue the same command to get the group chat id.
-3.  Start a conversation with your bot or invite him into a group you want to sent messages to.
+2.  A chat id for every chat you want to send messages to. Contact the [@myidbot](https://t.me/myidbot) bot and send the command `/getid` to get your personal chat id or invite it into a group and issue the same command to get the group chat id. Group IDs start with a hyphen, supergroup IDs start with `-100`.
+3.  Start a conversation with your bot or invite it into a group you want to sent messages to.
```
[the remark this answers is not in the document]
Comment (half-duplex) [reply health/notifications/telegram/README.md]: The bot was just down for a few days, it does seem like a good idea to mention the way that'll always work. Added, thanks!

[the remark this answers is not in the document]
Comment (thiagoftsm) [reply health/notifications/telegram/README.md]: All right, I am approving again this PR.
Thank you very much for your contribution.

**Pull request merged**